run_dir/design/flowcell.html

```diff
@@ -67,8 +79,27 @@
                         title="Treshold: {{ treshold }} million">{% for thousand in lane.get('clustersnb', '').split(',') %}<span class="thousand_group">{{ thousand }}</span>{% end %}
                     </td>
                     <th>%&nbsp;bases&nbsp;>&nbsp;Q30:</th>
-                    <td class="text-left {% if lane.get('overthirty', 0) < 30 %} danger {% elif lane.get('overthirty', 0) < 80 %}
-                      warning {% else %} success {% end %}">{{ lane.get('overthirty', 0) }}
+                    {% if flowcell.get('run_mode', '') == 'HiSeq X' %}
```

Comment:
yes, that would be better. i was just trying to avoid extra loops to save the time. in template we go through the data anyway. and in python not (because the data is handled in the status_db view).